README.md

```diff
@@ -56,3 +56,77 @@ $ python setup.py develop
 ```
 ./run_tests.sh
 ```
+
+## Running a Match
+- First of all, make sure that you placed your strategy class file inside the `game/` directory.
+- You can test your own strategy by playing against any of the built-in strategies or any other third party strategy by running a match.
+- To run a new match you should make use of the `run_game.py` script, also placed inside the `game/` directory
+- For this, open the file `run_game.py` on your favorite text editor (vim, for instance :P), and add your strategy class to the list of imports, just as the example below:
+```
+from my_super_duper_strategy import MySuperDuperStrategyClass
+```
+- After that, you'll need to add your strategy to the loockup_strategies dictionary, for which we recommend to follow the same convention:
+```
+loockup_strategies = {
+		(...)
+    'my_super_duper_strategy': MySuperDuperStrategy
```

Comment:
the keys in the lookup table don't have the work "strategy" at the end, maybe we should be consistent with that and leave it as `super_duper` (and remove the `my` everywhere)